elements/rh-card/src/rh-card.scss

```diff
@@ -14,114 +34,177 @@
   --rh-broadcasted--color--ui-link--hover:     #{rh-color(surface--accent--link--hover)};
   --rh-broadcasted--color--ui-link--focus:     #{rh-color(surface--accent--link--focus)};
+  // Header colors
+  .rh-card__header {
+    --rh-card__header--backgroundColor:         #{rh-color(surface--accent--harmony)};
+    --rh-broadcasted--color--text:              #{rh-color(surface--accent--harmony--text)};
+    --rh-broadcasted--color--ui-link:           #{rh-color(surface--accent--harmony--text)};
+    --rh-broadcasted--color--ui-link--visited:  #{rh-color(surface--accent--harmony--text)};
+    --rh-broadcasted--color--ui-link--hover:    #{rh-color(surface--accent--harmony--link--hover)};
+    --rh-broadcasted--color--ui-link--focus:    #{rh-color(surface--accent--harmony--link--focus)};
+  }
 }
 
 :host([size="small"]) {
   --rh-card--padding:        #{rh-var(container-spacer)};
 }
 
+// Remove top margin from typography inside the component
+@each $tag in (p, h1, h2, h3, h4, h5, h6) {
+  ::slotted(#{$tag}) {
+    margin-top: 0 !important;
+  }
+}
+
 .rh-card__header,
 .rh-card__body,
 .rh-card__footer {
   display: block;
-  margin: 0;
-}
-
-.rh-card__header::slotted(h1:first-child),
-.rh-card__header::slotted(h2:first-child),
-.rh-card__header::slotted(h3:first-child),
-.rh-card__header::slotted(h4:first-child),
-.rh-card__header::slotted(h5:first-child),
-.rh-card__header::slotted(h6:first-child) {
-  margin-top: 0 !important; // Specificity issue. Why?!
-  font-size: var(--rh-card_header--size);
-}
-
-.rh-card__body::slotted(*:nth-child(2)) { // child pseudo-selectors appear to be based on slotted code before upgrade.
-  margin-top: 0 !important;
+  &:not(:last-child) {
+    margin-bottom: var(--rh-card--spacing);
+  }
+}
+
+.rh-card__header {
+  --rh-broadcasted--color--text:              #{rh-color(surface--base--harmony--text)};
+  --rh-broadcasted--color--ui-link:           #{rh-color(surface--base--harmony--link)};
+  --rh-broadcasted--link--text-decoration:    none;
+  --rh-broadcasted--color--ui-link--visited:  #{rh-color(surface--base--harmony--link--visited)};
+  --rh-broadcasted--color--ui-link--hover:    #{rh-color(surface--base--harmony--link--hover)};
+  --rh-broadcasted--color--ui-link--focus:    #{rh-color(surface--base--harmony--link--focus)};
+  // Set the header background color
+  background-color: var(--rh-card__header--backgroundColor);
+  // Margin style only on top and sides
+  margin-top: calc(-1 * var(--rh-card--paddingTop));
+  margin-right: calc(-1 * var(--rh-card--paddingRight));
+  margin-left: calc(-1 * var(--rh-card--paddingLeft));
+  // Padding for the header region
+  padding-top: var(--rh-card--spacing); // Top padding differs if background color applied
+  padding-right: var(--rh-card--paddingRight);
+  padding-left: var(--rh-card--paddingLeft);
+  // Heading styles
+  @for $n from 1 to 6 {
+    &::slotted(h#{$n}:first-child) {
```

Comment:
It could be better to make these styles opt-in, by creating additional slots like this:

```
<slot class="rh-card__header" name="header">
  <slot class="rh-card__title" name="title"></slot>
  <slot class="rh-card__headline" name="headline"></slot>
</slot>
```

demo:
```
<rh-card>
  <header slot="header">
    <h2 slot="title">Features</h2>  <!-- opting into styles -->
    <h3>Check out all this cool stuff</h3> <!-- standard h3 styles from browser-->
  </header>
</rh-card>
```

alternatively could we just have a rh-standard-header component with 3-6 slots with various styles?

```
<rh-card>
  <rh-standard-header slot="header">
    <h2 slot="title">Features</h2>  <!-- opting into styles from rh-standard-header -->
  </rh-standard-header>
</rh-card>
```

Reply:
Good point! I was thinking more about the [general headline styles for cards](https://content.evernote.com/shard/s14/sh/fc7e701f-45fa-43ee-a080-b6d4d0ee9ce2/6ff68683dbfc9d72/res/32f9ef2f-061a-47e0-91c4-52c01397e35b/skitch.png) but you're right they don't all need to live in the header region. 

Just adding my notes from this morning's meeting, in keeping typography (content) styles within a type="content" component, should we need to support additional styles in the future, like this. 

```
    <rh-card color="lightest">
      <rh-header slot="rh-card__header">
        <h2 slot="rh-header__headline" variant="primary"><a href="#">Lightest card</a></h2>
      </rh-header>
 
      <rh-content>
      <p>This is the lightest rh-card and <a href="#">a link</a></p>
      </rh-content>

      <rh-cta slot="rh-card__footer" priority="primary"><a href="#">Learn more</a></rh-cta>
    </rh-card>
```

This example also keeps the slot names namespaced to the component, to avoid confusion of which header we're talking about.